app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,27 @@
+class SessionsController < ApplicationController
```

Comment:
If we decide to keep this PR purely for teacher CRUD-ing. We should move the session-related changes to a separate PR that deals with logins.